content/kubernetes/release-notes/7-2-4-releases/7-2-4-2.md

```diff
@@ -0,0 +1,164 @@
+---
+
+title: Redis Enterprise for Kubernetes release notes 7.2.4-2 (Aug 2023)
+linkTitle: 7.2.4-2 (Aug 2023)
+description: Support for Redis Enterprise 7.2.4, platform support updates, and bug fixes.
+weight: 10
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [ 
+    /kubernetes/release-notes/7-2-4-releases/7-2-4-2/,
+      ]
+---
+
+## Highlights
+
+This version offers:
+
+- Support for [Redis Enterprise 7.2.4]({{<relref "/rs/release-notes/rs-7-2-4-releases/rs-7-2-4-52">}})
+
+- New default engine, Speedb, for Auto Tiering (formerly Redis on Flash)
+
```

Comment:
Change "This version offers" to 
This release had the primary purpose of supporting the update of RS 7.2, which is a major Redis Enterprise Software update. As such, we reduced the scope of changes to support the underlying change of RS 7.2 and a few significant fixes. The following are the notable changes, except the regular update to the changing Kubernetes versions landscape.

Reply:
Changes approved by Brad via Slack